Support Nextjs + Stylable in a container

Hi, 

Following my last attempt to deploy nextJs + stylable with vercel - https://github.com/wix/stylable/issues/1710,
I'm now deploying the app using container and google cloud platform to host it.

I managed to run the app in a container with the styles working well,
The only issue is that the styles only apply when `NODE_ENV=development`.

if it's `NODE_ENV=production` the styles don't apply.

this are the scripts:

```
  "scripts": {
    "dev": "node server.js -p $PORT",
    "build": "next build",
    "start": "NODE_ENV=development node server.js -p $PORT"
  }
```

any idea how to combine  webpack in production env and apply the styles ?

all of the code can be found here - https://github.com/wix/stylable/issues/1710
if it's a burden that it's in another issue,  I'll copy/paste it here.

I updated to `next latest:  10.2.3` and added `stylable: 5.4.11` still getting this error:
```
(node:20) UnhandledPromiseRejectionWarning: Error: Cannot find module 'webpack/lib/dependencies/NullDependency'
Require stack:
- /app/node_modules/@stylable/webpack-plugin/cjs/stylable-dependencies.js
- /app/node_modules/@stylable/webpack-plugin/cjs/stylable-bootstrap-module.js
- /app/node_modules/@stylable/webpack-plugin/cjs/stylable-webpack-plugin.js
- /app/node_modules/@stylable/webpack-plugin/cjs/index.js
- /app/next.config.js
- /app/node_modules/next/dist/next-server/server/config.js
- /app/node_modules/next/dist/server/next.js
- /app/server.js
```

--- a/packages/webpack-plugin/src/webpack-config-stylable-excludes.ts
+++ b/packages/webpack-plugin/src/webpack-config-stylable-excludes.ts
@@ -6,8 +6,11 @@ export function applyWebpackConfigStylableExcludes(webpackConfig: Configuration)
 
 function insertStCssExclude(stRegex: RegExp) {
     return (_: string, value: RuleSetRule) => {
-        if (typeof value !== 'string' && value && value.test) {
-            if (value.test.toString().includes('css')) {
+        if (typeof value !== 'string' && value && (value.test || value.issuer)) {
+            if (
+                value.test?.toString().includes('css') ||
+                value.issuer?.toString().includes('css')
+            ) {
                 if (value.exclude && Array.isArray(value.exclude)) {
                     value.exclude.push(stRegex);
                     return false;

